sketchify/komand_sketchify/actions/sketchify/action.py

```diff
@@ -1,37 +1,28 @@
-import komand
-from .schema import SketchifyInput, SketchifyOutput
 # Custom imports below
 import requests
 
+import komand
+from komand.exceptions import PluginException
+from .schema import SketchifyInput, SketchifyOutput, Input, Output, Component
+
 
 class Sketchify(komand.Action):
 
     def __init__(self):
         super(self.__class__, self).__init__(
-                name='sketchify',
-                description='Turn a URL into a suspicious looking one',
-                input=SketchifyInput(),
-                output=SketchifyOutput())
+            name='sketchify',
+            description=Component.DESCRIPTION,
+            input=SketchifyInput(),
+            output=SketchifyOutput())
 
     def run(self, params={}):
-        url = params.get('url')
-        try:
-            r = requests.post('https://verylegit.link/sketchify', data = {'long_url': url})
-            r.raise_for_status()
-            sketchy_url = r.content
-        except Exception as e:
-            self.logger.error(e)
-            raise
-
-        return { 'url': sketchy_url.decode() }
-
-    def test(self):
+        url = params.get(Input.URL)
         try:
-            r = requests.post('https://verylegit.link/sketchify', data = {'long_url': 'test'})
+            r = requests.post('https://verylegit.link/sketchify', data={'long_url': url})
             r.raise_for_status()
             sketchy_url = r.content
         except Exception as e:
             self.logger.error(e)
-            raise
+            raise PluginException
```

Comment:
Need to log the error and the response text if possible. I don't think just logging the error will make it to the front end.